### PR TITLE
Add core dump docs for PHP in docker

### PR DESCRIPTION
**Pull request by realFlowControl**
### What does this PR do? What is the motivation?

We had a customer that experienced a crash but was unable to obtain a core dump as they were running PHP in docker, which is another layer to overcome when you need to create a core file.
This PR will add the docs to help customers that run PHP in docker create core files.

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/tracing/trace_collection/dd_libraries/php.md [hunk 1]
```diff
@@ -353,6 +353,25 @@ If no core dump was generated, check the following configurations and change the
 1. Ensure you have a suitable `ulimit` set in your system. You can set it to unlimited: `ulimit -c unlimited`.
 1. If your application runs in a Docker container, changes to `/proc/sys/*` have to be done to the host machine. Contact your system administrator to know the options available to you. If you are able to, try recreating the issue in your testing or staging environments.
 
+### Obtaining a core dump from within a docker container
```
Comment (buraizu) [inline content/en/tracing/trace_collection/dd_libraries/php.md:356]:
```suggestion
### Obtaining a core dump from within a Docker container
```

File: content/en/tracing/trace_collection/dd_libraries/php.md [hunk 2]
```diff
@@ -353,6 +353,25 @@ If no core dump was generated, check the following configurations and change the
...
+
+Obtaining a core dump in a docker container adds another layer of challenges to overcome. Here are a few tips to help you obtain a core dump:
```
Comment (buraizu) [inline content/en/tracing/trace_collection/dd_libraries/php.md:358]:
```suggestion
Use the information below to assist with obtaining a core dump in a Docker container:
```

File: content/en/tracing/trace_collection/dd_libraries/php.md [hunk 3]
```diff
@@ -353,6 +353,25 @@ If no core dump was generated, check the following configurations and change the
...
+
+1. The docker container needs to run as a privileged container and the ulimit for core files needs to be maxed out.
```
Comment (buraizu) [inline content/en/tracing/trace_collection/dd_libraries/php.md:360]:
```suggestion
1. The Docker container needs to run as a privileged container, and the `ulimit` value for core files needs to be set to its maximum as shown in the examples below.
```

File: content/en/tracing/trace_collection/dd_libraries/php.md [hunk 4]
```diff
@@ -353,6 +353,25 @@ If no core dump was generated, check the following configurations and change the
...
+   - In case you use `docker run` add the `--privileged` and the `--ulimit core=99999999999` argument
+   - In case you use `docker compose` add the following to the container in the `docker-compose.yml`:
```
Comment (buraizu) [inline content/en/tracing/trace_collection/dd_libraries/php.md:362]:
```suggestion
   - If you use the `docker run` command, add the `--privileged` and the `--ulimit core=99999999999` arguments
   - If you use `docker compose`, add the following to the `docker-compose.yml` file:
```

**Review by buraizu — CHANGES_REQUESTED** [review]
Thanks, just suggesting a few minor edits

**Comment by realFlowControl** [comment]
Hey there @buraizu 👋 
can you have another look and feel free to merge when everything is okay from your side

**Review by buraizu — APPROVED** [review]
@realFlowControl thanks for applying those changes, LGTM!

**Pull request merged**